pytest/lib/cluster.py

```diff
@@ -324,6 +324,11 @@ def start(self, boot_key, boot_node_addr):
     def kill(self):
         if self.pid.value != 0:
             os.kill(self.pid.value, signal.SIGKILL)
+            while True:
+                try:
+                    os.kill(pid, 0)
+                except OSError:
```

Comment:
Is it possible for it to return any other error?

Reply:
No, kill doesn't raise error if the pid exists, and raise OSError if pid doesn't exist